acceptance/acceptance_test.go

```diff
@@ -988,3 +995,85 @@ func prepareWheelBuildDirectory(t *testing.T, dir string) string {
 
 	return latestWheel
 }
+
+func applyBundleConfig(t *testing.T, tmpDir string, bundleConfig map[string]any) string {
+	validConfig := make(map[string]map[string]any, len(bundleConfig))
+
+	for _, configName := range utils.SortedKeys(bundleConfig) {
+		configValue := bundleConfig[configName]
+		if configValue == "" {
+			continue
+		}
+		// either "" or a map are allowed
+		cfg, ok := configValue.(map[string]any)
+		if !ok {
+			t.Fatalf("Unexpected type for BundleConfig.%s: %#v", configName, configValue)
+		}
+		validConfig[configName] = cfg
+	}
+
+	if len(validConfig) == 0 {
+		return ""
+	}
+
+	var configPath, configData string
+	filenames := []string{"databricks.yml", "databricks.yml.tmpl"}
```

Comment:
Why the tmpl file?

Also, the configuration could include the filename directly to make this not bundle-specific (or specific to `databricks.yml` but applicable to other YAML files in the test directory).

Reply:
Great idea, added BundleConfigTarget for this: https://github.com/databricks/cli/pull/2809/commits/3a991f453a325279f66ff525e782a661de194998